charts/registry-creds/templates/deployment.yaml

```diff
@@ -107,9 +107,12 @@ spec:
                   key: ACR_PASSWORD
             {{- end }}
           {{- range $key, $value := .Values.args }}
```

Comment:
This line should now go away (also doesn´t have corresponding end so I guess linting should fail)